boards/arduino-duemilanove/Kconfig

```diff
@@ -0,0 +1,16 @@
+# Copyright (c) 2020 HAW Hamburg
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+
+config BOARD_ARDUINO_DUEMILANOVE
+    bool
+    default y
+    select BOARD_COMMON_ARDUINO_ATMEGA
+    select CPU_MODEL_ATMEGA328P
```

Comment:
I think `CPU` and `CPU_MODEL` should be defined here, otherwise we are depending current `Makefile.features` for its definition. I think it should be checked that those match as well.

Reply:
As explained offline `CPU` is currently the entry point for Kconfig so still needs to be defined externally.